### PR TITLE
opencolorio: exported CMake wrapper, correct class name

### PR DESCRIPTION
**Pull request by p-podsiadly**
**opencolorio/1.1.1**

This PR adds a top level `CMakeLists.txt` wrapper, as suggested by @SpaceIm in the [previous PR](https://github.com/conan-io/conan-center-index/pull/2130#discussion_r455016464), so that it has similar layout to other recipes.

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/opencolorio/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_BINARY_DIR}")
```
Comment (uilianries) [inline recipes/opencolorio/all/CMakeLists.txt:7]:
```suggestion
```

I don't think it's required, once call, conan_basic_setup configures everything

File: recipes/opencolorio/all/conanfile.py [hunk 2]
```diff
@@ -11,7 +11,8 @@ class OpenEXRConan(ConanFile):
     options = {"shared": [True, False], "fPIC": [True, False]}
     default_options = {"shared": False, "fPIC": True}
     generators = "cmake", "cmake_find_package"
-    exports_sources = ["patches/*"]
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    short_paths = True
```
Comment (madebr) [inline recipes/opencolorio/all/conanfile.py:15]: @p-podsiadly Is this for fixing a build error on MSVC?
@uilianries @danimtb Poking you guys again.

Comment (uilianries) [reply recipes/opencolorio/all/conanfile.py:15]: Probably that same problem related to windows path in CCI, we need an internal fix for it.

Comment (madebr) [reply recipes/opencolorio/all/conanfile.py:15]: @p-podsiadly I think you can remove the `short_paths` now.

**Pull request merged**